src/rust/engine/src/context.rs

```diff
@@ -433,6 +433,7 @@ impl Core {
               err
             )
           })?;
+        // TODO: Use the once_cell::Lazy here
```

Comment:
This will be matched once per `pantsd` instance (on average), so probably not worth it.
```suggestion
```